scene/3d/baked_light_instance.cpp

```diff
@@ -1695,12 +1695,16 @@ void BakedLight::create_debug_mesh(DebugMode p_mode) {
 	MultiMeshInstance *mmi = memnew( MultiMeshInstance );
 	mmi->set_multimesh(mm);
 	add_child(mmi);
+#ifdef TOOLS_ENABLED
 	if (get_tree()->get_edited_scene_root()==this){
```

Comment:
@reduz Shouldn't you just expose `SceneTree::get_edited_scene_root()` for non-tools builds and have it return null? Every single time you add such a test you forget the `#ifdef TOOLS_ENABLED`, so I guess in your mind it's supposed to work already and return false for the condition.